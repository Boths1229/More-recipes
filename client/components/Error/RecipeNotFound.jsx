import React from 'react';

const RecipeNotFound = () => (
  <div className="container">
    <div className="row">
      <div className="col-md-12">
        <div className="not-found-container">
          <div className="text-center error-margin">
            <div>
              <i className="fa fa-exclamation-circle fa-5x" />
            </div>
            <h1>You do not have any recipe in your catalog </h1>
          </div>
        </div>
      </div>
    </div>
  </div>
);

export default RecipeNotFound;
