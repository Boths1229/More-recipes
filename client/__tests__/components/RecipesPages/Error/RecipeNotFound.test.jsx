import React from 'react';
import { shallow } from 'enzyme';
import RecipeNotFound from '../../../../components/Error/RecipeNotFound';

describe('IndexHeader component', () => {
  it('Recipe not found component', () => {
    const output = shallow(<RecipeNotFound />);
    expect(output).toMatchSnapshot();
  });
});
