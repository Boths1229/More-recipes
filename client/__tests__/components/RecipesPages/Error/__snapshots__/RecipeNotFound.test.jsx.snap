// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexHeader component Recipe not found component 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <div
        className="not-found-container"
      >
        <div
          className="text-center error-margin"
        >
          <div>
            <i
              className="fa fa-exclamation-circle fa-5x"
            />
          </div>
          <h1>
            You do not have any recipe in your catalog 
          </h1>
        </div>
      </div>
    </div>
  </div>
</div>
`;
